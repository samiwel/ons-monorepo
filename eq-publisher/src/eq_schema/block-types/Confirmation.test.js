const Confirmation = require("./Confirmation");

describe("Confirmation", () => {
  it("should build valid runner Confirmation", () => {
    const confirmation = new Confirmation();
    expect(confirmation).toMatchSnapshot();
  });
});
