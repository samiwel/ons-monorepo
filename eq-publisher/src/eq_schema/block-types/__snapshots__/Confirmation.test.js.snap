// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Confirmation should build valid runner Confirmation 1`] = `
Confirmation {
  "blocks": Array [
    Object {
      "description": "",
      "id": "confirmation",
      "questions": Array [
        Object {
          "guidance": Object {
            "content": Array [
              Object {
                "list": Array [
                  "You will not be able to access or change your answers on submitting the questionnaire",
                  "If you wish to review your answers please select the relevant completed sections",
                ],
              },
            ],
          },
          "id": "ready-to-submit-completed-question",
          "title": "Submission",
          "type": "Content",
        },
      ],
      "title": "You are now ready to submit this survey",
      "type": "Confirmation",
    },
  ],
  "id": "confirmation-group",
  "title": "confirmation",
}
`;
